Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/groovy
+node {
+    env.http_proxy="http://w3p2.atos-infogerance.fr:8080"
```

Comment:
May this could be provide by a variable environment (it's infrastructure dependent).